### ISSUE TITLE
Restrict access to some folders

### ISSUE THREAD

**Issue by husnaintariq01** [idleberg/Bootstrap-Listr#348]
Respected Sir,

I have one concern. I am using your plugin and I would like to know if a certain tweak can be performed on it. My website url is like this:

example.com/ROOT/Sub_directories/manyfiles

The **Root** folder is the folder where I have uploaded the lister plugin. It is working fine.

What I want to do is restrict access to this ROOT folder. Like I can give link to people of **Sub Directories** and not worry about them to click on the home icon and view all of the folders. The reason why I can't use plugin again for all subfolders is that it will make the server heavy and there are many subfolders like in hundreds so it is very difficult.

In a nutshell:

Can I restrict access to my root folder? It will be great if I can just redirect it or simply just display forbidden error.
Right now I am using 403 redirect but it just wont work. 

Please help me Sir @idleberg. Thanks a lot.

**Comment by idleberg** [comment]
That's the job of your webserver
